tests/test_accessibility.py

```diff
@@ -108,6 +112,196 @@ def test_axs_audit_limit_rules(self, mock_rules):
 
         self.assertEqual(0, len(result.warnings))
 
+    @patch('tests.pages.AccessibilityPage.axs_audit_rules_to_run')
+    def test_axs_audit_run_multiple_rules(self, mock_rules):
+        # Tests that other all errors are reported if checking more than one.
```

Comment:
nit: typo...I think you want "...that all other errors..." instead of "...that other all errors..."